internal/build-utils/package.json

```diff
@@ -21,6 +28,6 @@
     "consola": "^2.15.3"
   },
   "devDependencies": {
-    "unbuild": "^0.7.4"
```

Comment:
`unbuild@2.0` has some incompatible updates in my mind, which I recommend not mixing into vitepress PR.

Reply:
If I were not to upgrade `unbuild`, `npm run docs:dev` would fail.
Here is why: After upgrading, VitePress uses `vite@5.x` under the hood, which only supports ESM. And `unbuild@0.7.4 --stub` outputs a non-standard `.mjs` file that cannot be consumed correctly by ESM.

Reply:
Ok, then let's move forward.
@jw-foss is working on pnpm monorepo about `utils` and `constants`.
Do you have any other suggestions?

Reply:
I have no idea.🤔  Is there any connection between them? Upgrading `unbuild` doesn't seem to affect the work you mentioned.